### PR TITLE
Move `PreferencesHomeFeed` to a screen instead of a modal

### LINKED ISSUE #1014: Unable to scroll in Home Feed Preferences
**Describe the bug**
User is unable to scroll down and fully display all home feed Preferences settings. 

Resolve issue by toggling any visible setting or changing any visible slider 

**To Reproduce**

1. Open bluesky app
2. From the app home screen select hamburger menu in the top left
3. Select Settings 
4. Select Home Feed Preferences 
5. Issue occurs here and shows as user being unable to scroll down and fully display all home feed Preferences 
6. Resolve issue by toggling any visible setting or changing any visible slider

**Expected behavior**

The home screen Preferences screen should either display fully from the start, or allow the user to scroll it without first needing to toggle an option. 

**Screenshots**

Screenshot displaying the home feed Preferences screen not displaying fully and not allowing the user to scroll down
Screenshot displaying the home feed Preferences screen not displaying fully and not allowing the user to scroll down

creenshot displaying the home feed Preferences screen after user is able to scroll down 
Screenshot displaying the home feed Preferences screen after user is able to scroll down.

[Screen recording hosted on Google drive](https://photos.app.goo.gl/72WzEi7vtgrrrLYj6)

Note that when recording the app behaves as expected when opening the home feed preferences section the first time, but shows the bug after backing out and then reopening it.

I do not know why the behavior requires a back out to trigger when screen recording, but occurs every time when not screen recording. 

**Details**

 - Platform: Android
 - Platform version: 13
 - App version: 1.41.0

**Additional context**

Device in use is a Google Pixel 7.
On my device, under Accesability Settings - > Display Size and Text
Font size set to option 3
Display size set to option 2

Under accessibility Bitwarden is allowed to enter usernames and passwords, and a bitwarden bubble is displayed in the screen recording.

Potentially similar to [this closed issue](https://github.com/bluesky-social/social-app/issues/901)

### PR DESCRIPTION
**Pull request by ansh**
Fix #1014 forever (it is a non-deterministic bug so moving to a screen was the best option)

web
CleanShot 2023-08-30 at 11 38 00@2x

### CODE REVIEW

**Review by estrattonbailey — APPROVED** [review]
LGTM

**Review by pfrazee — CHANGES_REQUESTED** [review]
CleanShot 2023-08-29 at 17 22 16@2x

On Web, can we get the borders on the side and also update the "Done" button so it's not so massive?

**Comment by pfrazee** [comment]
Tests are failing for some reason, maybe need a mock update

File: bskyweb/cmd/bskyweb/server.go [hunk 1]
```diff
@@ -168,6 +168,7 @@ func serve(cctx *cli.Context) error {
 	e.GET("/moderation/blocked-accounts", server.WebGeneric)
 	e.GET("/settings", server.WebGeneric)
 	e.GET("/settings/app-passwords", server.WebGeneric)
+	e.GET("/settings/preferences-home-feed", server.WebGeneric)
```
Comment (pfrazee) [inline bskyweb/cmd/bskyweb/server.go:171]: Can we tune this path up a bit? maybe `/settings/home-feed` or at least `/settings/home-feed-preferences`

**Comment by pfrazee** [comment]
Looks great on mobile, just need to fix up web, the tests, and (unless 👎 ) update the path

File: bskyweb/cmd/bskyweb/server.go (hunk 1 above)
Comment (ansh) [reply bskyweb/cmd/bskyweb/server.go:171]: I tried to keep the naming consistent with what is was before but I can change it

Comment (ansh) [reply bskyweb/cmd/bskyweb/server.go]: Done

**Review by pfrazee — APPROVED** [review]
:fire: solid, thanks for the changes

**Pull request merged**